src/backend/alembic/versions/2024_08_21_e284b705fb23_.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
```

Comment:
What happened with this migration? Did we have to merge heads for `('0091b3b3b4dc', 'c301506b3676')` ?

Reply:
Can you try deleting your migrations, pulling from main, `make reset-db` and remake your migrations

Reply:
this was the issue i believe
https://stackoverflow.com/a/51456404

Reply:
let me try doing what u said and see if it works

Reply:
that fixed it, ty